lib/common/schemaUtilsCommon.js

```diff
@@ -4,6 +4,28 @@
 
 const parse = require('../parse.js');
 
+/**
+ *  Remove the # character from the beginning of a schema path
+ * @param {string} schemaPath - a defined schemaPath
+ * @returns {string} - the schema path with # removed
+ */
+function removeSharpAndSlashFromFirstPosition(schemaPath) {
```

Comment:
@LuisTejedaS Is it be possible that schemaPath doesn't have `#` in start?

Reply:
Yes, it will only be removed if is present

Reply:
@LuisTejedaS We are removing `#/` with `schemaPath.slice(2)` but when `#` is not present we are not removing anything so is it possible that we can have something like `/path/to/something`?

Reply:
Yes, for example /id should be converted into .id so when starts with / we do not remove it